src/panels/beaconSummaryPanel/BeaconSummaryEditing.tsx

```diff
@@ -121,13 +121,41 @@ export const BeaconSummaryEditing: FunctionComponent<{
                       }
                     />
 
+                    <TabulatedRow
+                      displayKey={
+                        <label htmlFor="beaconType">
+                          <Typography>
+                            {"Beacon type" + WritingStyle.KeyValueSeparator}
+                          </Typography>
+                        </label>
+                      }
+                      value={
+                        <Field as="select" name="beaconType">
+                          <option value="" label={Placeholders.NoData} />
+                          {beaconTypes.map((beaconType: string) => {
```

Comment:
```suggestion
                          {Object.values(BeaconTypes).map((beaconType: string) => {
```

Thinking maybe here we could reuse this [BeaconTypes enum](https://github.com/mcagov/beacons-backoffice/blob/c8521082f302a453f633c2c2d59ee8ca83dcd378/src/entities/IBeacon.ts#L38) which we use on [Legacy beacons](https://github.com/mcagov/beacons-backoffice/blob/c8521082f302a453f633c2c2d59ee8ca83dcd378/src/entities/ILegacyBeacon.ts#L11). Not sure if there's any reason not to 🤔